src/Microsoft.VisualStudio.Services.Agent/ProcessInvoker.cs

```diff
@@ -196,4 +227,15 @@ private void OnOutputDataReceived(object sender, DataReceivedEventArgs e)
             }
         }
     }
+
+    public sealed class ProcessExitCodeException : Exception
```

Comment:
I had my eye on GitCommandManager when I added this although I didn't touch that class. I still had a use for the exception in TeeTFCommandManager.